src/translations/translations.fr.js

```diff
@@ -592,6 +592,34 @@ export const translations = {
   //#region cc-select
   'cc-select.required': `obligatoire`,
   //#endregion
+  //#region cc-ssh-key-list
+  'cc-ssh-key-list.add.btn': `Ajouter la clé`,
+  'cc-ssh-key-list.add.info': () => sanitize`<p>Vous devez associer une clé SSH à votre compte si vous désirez déployer via Git. Utilisez ce formulaire à cet effet.</p><p>Vous pouvez créer une clé SSH avec la commande suivante&nbsp;:</p><code>ssh-keygen -t ed25519 -C "my-email@example.com"</code><p>La clé publique générée est sauvegardée dans le fichier "*.pub".</p>`,
+  'cc-ssh-key-list.add.name': `Nom`,
+  'cc-ssh-key-list.add.public-key': `Clé publique`,
+  'cc-ssh-key-list.add.title': `Ajouter une nouvelle clé`,
+  'cc-ssh-key-list.doc.info': () => sanitize`Pour plus d'aide, vous pouvez consulter notre <a href="https://www.clever-cloud.com/doc/admin-console/ssh-keys/">documentation</a> (en anglais).`,
```

Comment:
very small details:
It would be better if the "en Anglais" mention was part of the link text. This way, screen reader users don't have to check the context of the link.
Anglais should start with a cap because it's a noun here and not an adjective ("les utilisateurs anglais" vs "en Anglais, on dit ça").